pages/common/git-tag.md

```diff
@@ -0,0 +1,19 @@
+# git tag
+
+> Create, list, delete or verify tags
+
+- List all tags
+
+`git tag`
+
+- Create a named tag pointing to the current commit
+
+`git tag {{tag_name}}`
+
+- Create a tag with the given message
```

Comment:
I'm asking about what the purpose of `{{tag_message}}`, i.e. how it's used after the tag is created. Is it supposed to merely appear in git log as a summary of that history entry, or is it usually meant to be a more public, detailed description of the tag / release?